pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator_test.go

```diff
@@ -196,40 +196,20 @@ func TestIsFSSEnabledInSV(t *testing.T) {
 
 // TestIsFSSEnabledInVanilla tests IsFSSEnabled in vanilla flavor - all scenarios
 func TestIsFSSEnabledInVanilla(t *testing.T) {
-	internalFSS := map[string]string{
-		"csi-migration": "true",
-		"volume-extend": "false",
-		"volume-health": "disabled",
-	}
 	internalFSSConfigMapInfo := FSSConfigMapInfo{
 		configMapName:      cnsconfig.DefaultInternalFSSConfigMapName,
 		configMapNamespace: cnsconfig.DefaultCSINamespace,
-		featureStates:      internalFSS,
 		featureStatesLock:  &sync.RWMutex{},
 	}
 	k8sOrchestrator := K8sOrchestrator{
-		clusterFlavor: cnstypes.CnsClusterFlavorVanilla,
-		internalFSS:   internalFSSConfigMapInfo,
-	}
-	// Should be enabled
-	isEnabled := k8sOrchestrator.IsFSSEnabled(ctx, "csi-migration")
-	if !isEnabled {
-		t.Errorf("csi-migration feature state is disabled!")
-	}
-	// Should be disabled
-	isEnabled = k8sOrchestrator.IsFSSEnabled(ctx, "volume-extend")
-	if isEnabled {
-		t.Errorf("volume-extend feature state is enabled!")
-	}
-	// Wrong value given
-	isEnabled = k8sOrchestrator.IsFSSEnabled(ctx, "volume-health")
-	if isEnabled {
-		t.Errorf("volume-health feature state is enabled even when it was assigned a wrong value!")
+		clusterFlavor:      cnstypes.CnsClusterFlavorVanilla,
+		internalFSS:        internalFSSConfigMapInfo,
+		releasedVanillaFSS: getReleasedVanillaFSS(),
 	}
 	// Feature state missing
-	isEnabled = k8sOrchestrator.IsFSSEnabled(ctx, "online-volume-extend")
+	isEnabled := k8sOrchestrator.IsFSSEnabled(ctx, "unknown-performance-feature")
```

Comment:
What is `unknown-performance-feature` ? Why do we need this ?

Reply:
The test used to provide 3 known fss'es as data and test for a 4th fss. Since we have hardcoded all enabled fss, to still test for an unknown fss value, I added this one.